packages/react-native-skia/components/image/blurRadius.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { View, AppRegistry, Image, Text } from 'react-native';
+var resizemodes = [
+    "center",
+    "cover",
+    "contain",
+    "stretch",
+    "repeat",
+];
+var blurRadius =[0,1,3,5,10,50,100,200,2000]
```

Comment:
1. please also check negative values and when no blurRadius value is been set
2. check for jpg images as well
3. check with no shadow 
4. app is not checking tests with resizeMode